### ISSUE TITLE
Proper using of execSQL() when we don't care about rows affected

### ISSUE THREAD

**Issue by whitebyte** [mysql-d/mysql-native#60]
```D
// ...
string query = "DROP TABLE `users`";
auto cmd = new Command(c, query);
uint rowsAffected; // <-- don't need it!
cmd.execSQL(rowsAffected);
```

There are situations when we absolutely don't care about how much rows affected by our query. What is the proper way of using execSQL() without creating redunant variable?

**Comment by Abscissa** [comment]
Although I haven't tried it, you might be able to do:

```d
cmd.execSQL(uint ra);
```

If not, you can just make a wrapper:

```d
void execSQL(Command cmd) {
     uint ra;
     cmd.execSQL(ra);
}
//...
cmd.execSQL();
```

But that wrapper really should be added to mysql-native itself.

**Comment by whitebyte** [comment]
```d
cmd.execSQL(uint ra);
```

Unfortunately this string seems to be illegal in D.

Currently I'm using a simple overloaded method:

```d
    
    // right after original execSQL in commands.d
    bool execSQL()
    {
        scope(failure) _con.kill();

        _con.sendCmd(CommandType.QUERY, _sql);
        _fieldCount = 0;
        ubyte[] packet = _con.getPacket();
        bool rv;
        if (packet.front == ResultPacketMarker.ok || packet.front == ResultPacketMarker.error)
        {
            _con.resetPacket();
            auto okp = OKErrorPacket(packet);
            enforcePacketOK(okp);
            _con._serverStatus = okp.serverStatus;
            _insertID = okp.insertID;
            rv = false;
        }
        else
        {
            // There was presumably a result set
            assert(packet.front >= 1 && packet.front <= 250); // ResultSet packet header should have this value
            _headersPending = _rowsPending = true;
            _pendingBinary = false;
            auto lcb = packet.consumeIfComplete!LCB();
            assert(!lcb.isNull);
            assert(!lcb.isIncomplete);
            _fieldCount = cast(ushort)lcb.value;
            assert(_fieldCount == lcb.value);
            rv = true;
        }
        return rv;
    }
```

It works for me, so technically this issue may be closed. Are there better ways of resolving this? should I make PR for this?

**Comment by Abscissa** [comment]
There's no need to duplicate the entire function, just a wrapper will do and be vastly less problematic to maintain. I'll add one now.

**Comment by Abscissa** [comment]
Fixed and tagged v0.1.3
